%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
% Medium Length Graduate Curriculum Vitae
% LaTeX Template
% Version 1.1 (9/12/12)
%
% This template has been downloaded from:
% http://www.LaTeXTemplates.com
%
% Original author:
% Rensselaer Polytechnic Institute (http://www.rpi.edu/dept/arc/training/latex/resumes/)
%
% Important note:
% This template requires the res.cls file to be in the same directory as the
% .tex file. The res.cls file provides the resume style used for structuring the
% document.
%
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

%----------------------------------------------------------------------------------------
% PACKAGES AND OTHER DOCUMENT CONFIGURATIONS
%----------------------------------------------------------------------------------------

\documentclass[overlapped, line, 10pt]{res} % Use the res.cls style, the font size can be changed to 11pt or 12pt here

\usepackage{helvet} % Default font is the helvetica postscript font
\usepackage{hyperref}
\usepackage{natbib}  % natbib is used in order to include DOIs in the reference information
\usepackage{doi}
%\usepackage{newcent} % To change the default font to the new century schoolbook postscript font uncomment this line and comment the one above

% This is used to remove indentation within the itemize environment (based on https://tex.stackexchange.com/questions/91124/itemize-removing-natural-indent#91128)
\usepackage{enumitem}
\setlist[itemize]{leftmargin=*}
% Also replace itemize dots with dashes:
\renewcommand\labelitemi{$-$}

\setlength{\textwidth}{5.5in} % Text width of the document

% Makes the random asterisk (*), that appears above the bibtex references for some reason, disappear.
% See http://tex.stackexchange.com/questions/28339/random-asterisk-when-using-bibtex-and-res-cls
\makeatletter
\renewenvironment{thebibliography}[1]
     {\list{\@biblabel{\@arabic\c@enumiv}}%
           {\settowidth\labelwidth{\@biblabel{#1}}%
            \leftmargin\labelwidth
            \advance\leftmargin\labelsep
            \@openbib@code
            \usecounter{enumiv}%
            \let\p@enumiv\@empty
            \renewcommand\theenumiv{\@arabic\c@enumiv}}%
      \sloppy
      \clubpenalty4000
      \@clubpenalty \clubpenalty
      \widowpenalty4000%
      \sfcode`\.\@m}
     {\def\@noitemerr
       {\@latex@warning{Empty `thebibliography' environment}}%
      \endlist}
\makeatother

% This is necessary to get citation keys when natbib is used
% (natbib is used in order to include DOIs in the reference information)
% Source: (secondary) https://tex.stackexchange.com/questions/4158/list-the-citation-key-shown-by-citep-in-the-references
% and (primary) http://newsgroups.derkeiler.com/Archive/Comp/comp.text.tex/2008-07/msg00540.html
\makeatletter
\def\@lbibitem[#1]#2{%
\if\relax\@extra@b@citeb\relax\else
\@ifundefined{br@#2\@extra@b@citeb}{}{%
\@namedef{br@#2}{\@nameuse{br@#2\@extra@b@citeb}}}\fi
\@ifundefined{b@#2\@extra@b@citeb}{\def\NAT@num{}}{\NAT@parse{#2}}%
\item[\hfil\hyper@natanchorstart{#2\@extra@b@citeb}\citep{#2}%
\hyper@natanchorend]%
\NAT@ifcmd#1(@)(@)\@nil{#2}}
\makeatother

\begin{document}

%----------------------------------------------------------------------------------------
% NAME AND ADDRESS SECTION
%----------------------------------------------------------------------------------------

\name{Alexej Gossmann}

\address{
  Web: \href{http://www.alexejgossmann.com/}{alexejgossmann.com} \textbar
  LinkedIn: \href{https://www.linkedin.com/in/alexejgossmann/}{alexejgossmann} \textbar
  Github: \href{https://github.com/agisga}{agisga} \textbar
  Email: \href{mailto:agossman@tulane.edu}{agossman@tulane.edu}
}

%----------------------------------------------------------------------------------------

\begin{resume}

%---

%\section{OBJECTIVE}

%---

\section{AREAS OF INTEREST}

Topics adjacent to the performance evaluation of artificial intelligence (AI) / machine learning (ML) enabled devices or software in medicine.
% Topics adjacent to performance evaluation in medical AI/ML, including test data reuse, domain generalization, and other generalization issues, imperfect reference standard, and other sources of bias in validation studies of AI/ML algorithms in medicine.
%Development of statistical and machine learning techniques for feature selection and prediction on big high-dimensional datasets, with focus on understanding how false research findings can arise, and how they can be avoided;
%applications in genomics and neuroimaging, and other medical applications of machine learning;
%translation of the statistical and machine learning methodology into a useful (software) product.

%---

\section{SKILLS}

{\sl Core:} Statistics, machine learning, artificial intelligence, data science, (applied) mathematics, scientific research.\\
{\sl Software, programming, tools:} Python, R, C, C++, Ruby, Matlab, \LaTeX, Linux/Unix, Git and Github, HTML, CSS, Vim, AWS and Google Cloud.\\
{\sl Domain knowledge:} Medical imaging, genomics.\\
{\sl Language knowledge:} German, Russian, English.

%---

\section{EDUCATION}

{\sl PhD,} Bioinnovation \\
Tulane University, New Orleans, Louisiana, August 2018\\
%Dissertation: Regaining control of false findings in feature selection, classification, and prediction on neuroimaging and genomics data\\
GPA: 3.967

{\sl Doctoral research,} Mathematics (PhD candidacy, all but dissertation, withdrawn)\\
Tulane University, New Orleans, Louisiana, through January 2017\\
GPA: 3.978

{\sl MS,} Statistics \\
Tulane University, New Orleans, Louisiana, May 2014\\
GPA: 3.975

{\sl BS,} Mathematics\\
Technische Universit\"{a}t Darmstadt, Darmstadt, Germany, May 2012\\
%Thesis: On disjunction and numerical existence properties of extensions of Heyting arithmetic (supervised by Dr. Ulrich Kohlenbach)\\
GPA: 3.7 (1.54 German grade)

{\sl Workshops and summer schools}\\
IPAM New Deep Learning Techniques, Los Angeles, CA, February 2018.~$|$
SAMSI Distributed and Parallel Data Analysis Workshop, September 2016.~$|$
SAMSI Optimization Opening Workshop, August 2016.~$|$
SAMSI Optimization Summer School, August 2016.~$|$
2nd Summer Institute in Statistics for Big Data, University of Washington, Seattle, June 2016.~$|$
21st Summer Institute in Statistical Genetics, University of Washington, Seattle, WA, June 2016.~$|$
20th Summer Institute in Statistical Genetics, University of Washington, Seattle, WA, June 2015.~$|$
SAMSI Industrial Mathematical and Statistical Modeling Workshop for Graduate Students, June 2014.

%---

\section{EXPERIENCE}

\title{Staff Fellow, Mathematical Statistician}
\employer{Division of Imaging, Diagnostics, and Software Reliability (CDRH/OSEL/DIDSR) at the U.S. Food \& Drug Administration}
\dates{Sep 2018 -- Present}
\location{FDA, Silver Spring, MD}
\begin{position}
  \begin{itemize}
    \item Scientific research and regulatory review activities related to the evaluation of medical devices with an AI, ML, or Big Data component in the Division of Imaging, Diagnostics, and Software Reliability (FDA/CDRH/OSEL/DIDSR).
    \item Coordinator of the ``Fundamental AI'' research sub-program at FDA/CDRH/OSEL.
    \item Mentorship of research fellows and interns.
    \item Publications in Biometrics, JAMIA, SIMODS, UAI, SPIE MI, MED-NeurIPS, etc.
    \item Invited presentations at PIcc23, REdI 2023, MICCAI 2022 tutorial, DIA Europe 2022, Xavier AI Experts Forum 2021, KDD 2021 workshop, etc.
    \item Participation in regulatory review of over 100 submissions (510(k), De Novo, Q-submissions) of AI/ML-based medical devices/software; and participation in the development of regulatory guidance documents related to medical AI/ML.
  \end{itemize}
\end{position}

\title{Research assistantship (doctoral research)}
\employer{The Multiscale Bioimaging and Bioinformatics Laboratory}
\dates{Jan 2015 -- Jul 2018}
\location{Tulane University, New Orleans, LA}
\begin{position}
  Research in statistics and machine learning with application in genomics and neuroimaging under the supervision of Dr. Yu-Ping Wang, resulting in six peer-reviewed publications (\cite{gossmann2015, cao2015BCB, cao2015bioinformatics, Gossmann2017-yu, Gossmann2017-ln, brzyski2016}), presentations at multiple conferences and workshops, and several open source software packages (developed in R and C++).
\end{position}

\title{Student intern (ORISE)}
\employer{Division of Imaging, Diagnostics, and Software Reliability (CDRH/OSEL/DIDSR) at the U.S. Food \& Drug Administration}
\dates{May -- Aug 2017, and Jan -- Feb 2018}
\location{FDA, Silver Spring, MD}
\begin{position}
  Machine learning research (with software implementation in R) resulting in a conference presentation and associated publication \cite{gossmann2018} related to the evaluation of machine learning systems in medicine.
\end{position}

\title{Student developer}
\employer{Google Summer of Code 2015}
\dates{May -- August 2015}
\location{Remote position}
\begin{position}
  Project: Adding Linear Mixed Effects Models Support to SciRuby (supervised by Pjotr Prins from the Ruby Science Foundation). The created statistical software package (implemented in Ruby) received about 2700 downloads (\url{https://rubygems.org/gems/mixed_models}), and was considered for deployment at \url{http://www.genenetwork.org}.
\end{position}

\title{Research assistantship}
\employer{Department of Mathematics}
\location{Tulane University, New Orleans, LA}
\dates{Jun -- Aug 2014}
\begin{position}
  Worked under the supervision of Dr. Oleksandr Gromenko on statistical methods for the analysis of spatio-temporal processes with software implementation in R and C++, and with application in weather prediction (results unpublished).
\end{position}

\title{Statistics MS research project}
\employer{Department of Mathematics}
\location{Tulane University, New Orleans, LA}
\dates{Sept 2013 -- May 2014}
\begin{position}
  Analysis of bone growth data using mixed-effects smoothing spline ANOVA methods (supervised by Dr. Michelle Lacey) with data analyses performed in R, resulting in a journal publication \cite{sammarco2015}.
\end{position}

\title{Academic Mentor}
\employer{Tulane University, New Orleans, LA}
\dates{Oct 2017 -- May 2018}
\begin{position}
  Design, mentoring, and guidance of an undergraduate research project applying machine learning methods to a large neuroimaging-genomics dataset, resulting in two presentations by the mentored undergraduate student, entitled \emph{``Exploratory Analysis and Predictive Modeling of Neurodevelopmental Phenotypes from fMRI Data''} at the 2018 Health Sciences Research Days at Tulane University, and at the 2018 Tulane School of Science and Engineering Poster Days (top 3 finalist in the poster competition).
\end{position}

\title{Co-Mentor}
\employer{Google Summer of Code 2016 (with the Ruby Science Foundation)}
\location{Remote position}
\dates{May -- August 2016}
\begin{position}
  Involved with project proposals, selection/interviewing of students, mentoring. Mentored project: Categorical data support for Daru, Statsample and Statsample-glm. The mentored student (Lokesh Sharma) made major improvements to several open source software packages for data analysis in Ruby.
\end{position}

\title{Co-Mentor}
\employer{Google Summer of Code 2017 (with the Ruby Science Foundation)}
\location{Remote position}
\dates{May -- August 2017}
\begin{position}
  Involved with project proposals, selection/interviewing of students, mentoring. Mentored project: Creating the fastest math libraries for Ruby by using the GPU through OpenCL and ArrayFire. Outcome: a Ruby library for scientific computing on the GPU (\url{https://github.com/arrayfire/arrayfire-rb}) and multiple conference presentations by the mentored student (Prasun Anand).
\end{position}

\title{Instructor, co-teacher, teaching assistant}
\employer{Technische Universit\"{a}t Darmstadt, and Tulane University}
\location{Darmstadt, Germany, and New Orleans, LA}
\dates{Fall semester 2010 -- Fall semester 2014}
\begin{position}
  Instructor for Calculus 1, co-teacher for Statistics for Scientists, and teaching assistant for Real Analysis 1 and 2 as well as for various undergraduate courses in mathematics and statistics.
\end{position}

%---

\section{PUBLICATIONS (sorted alphabetically)}

\nocite{*}
\bibliographystyle{plainnat}  % citations are in alphabetic order
%\bibliographystyle{unsrtnat}  % citations are in the same order as in the bibtex file --> this doesn't work because some references are cited above, and thus, for example, first cited paper appears first in the bibliography here
\bibliography{Alexej_Gossmann_CV}

%---

\section{CONFERENCE PRESENTATIONS}

\begin{itemize}
  \item Gossmann, A., A Regulatory Perspective on AI/ML-enabled Medical Devices, with Consideration of Real-World Data and Real-World Evidence;
    Invited presentation at \textit{Europe DIA 2022 Panel on Enabling AI in Healthcare -- the Gaps between State of Science and Regulatory Standards}, April 2022.
  \item Gossmann, A., Issues in reference standard determination for performance evaluation studies of AI/ML-based medical devices;
    Invited presentation for the \textit{Xavier AI Experts forum}, December 2021.
  \item Gossmann, A., Performance evaluation and regulatory considerations for AI/ML in medical imaging;
    Invited talk presented at \textit{KDD 2021 Health Day and 2021 KDD Workshop on Applied Data Science for Healthcare}, August 2021.
  \item
    Malik B., Cha K. H., Sahiner B., Gossmann, A., Software Pipelines for AI-Based Radiologic Image Analysis;
    Poster presented by Malik B. (summer intern student) at the \textit{2021 FDA Science Forum}.
  \item 
    Adkins-Chatfield T. A., Gossmann A., Image-to-Image Translation Using CycleGAN of Breast Cancer Tissue Slides Stained with an Antibody against HER2/NEU;
    Poster presented by Adkins-Chatfield T. A. (summer intern student) at \textit{the Annual Biomedical Research Conference for Minority Students (ABRCMS) 2020}.
  \item
    Gossmann A., Cha K. H., Sun X., Performance deterioration of deep neural networks for lesion classification in mammography due to distribution shift: an analysis based on artificially created distribution shift;
    Contributed talk presented by Cha K. H. (substitute due to travel complications of lead author) at the \textit{SPIE Medical Imaging Medical Imaging 2020: Computer-Aided Diagnosis},
    February 2020, Houston, TX.
  \item Gossmann A., Cha K. H., Sun X., Variational inference based assessment of mammographic lesion classification algorithms under distribution shift;
    Poster presented at \textit{Medical Imaging Meets NeurIPS Workshop (MED-NeurIPS) 2019},
    December 2019, Vancouver, Canada.
  \item
    Gossmann A. Evaluation of continuously learning AI/ML-enabled medical devices.
    Invited talk presented at \textit{Next Generation Dx},
    August 2019, Washington, DC.
  \item
    Gossmann A., Chen W., Sahiner, B., Assessment of Classifier Performance Using a Reference Classifier with Known Performance and an Unlabeled Dataset;
    (Abstract/Program \#307008).
    Contributed talk presented at the \textit{Joint Statistical Meetings},
    July 28, 2019, Denver, CO.
  \item
    Gossmann A., Pezeshk, A., Sahiner, B., Test data reuse for evaluation of adaptive machine learning algorithms: over-fitting to a fixed ``test'' dataset and a potential solution;
    (Paper 10577-19).
    Contributed talk presented at the \textit{SPIE Medical Imaging symposium, Image Perception, Observer Performance, and Technology Assessment conference},
    February 11, 2018, Houston, TX.
  \item
    Gossmann A., Wang Y.-P., SlopeCCA and gslopeCCA: sorted L1 penalized canonical correlation analysis;
    (Abstract/Program \#2803W).
    Poster presented at the \textit{66th Annual Meeting of The American Society of Human Genetics},
    October 19, 2016, Vancouver, Canada.
  \item
    Gossmann A., Cao S., Wang Y.-P., Identification of Significant Genetic Variants via SLOPE, and its Extension to Group SLOPE;
    (Abstract/Program \#1343F).
    Poster presented at the \textit{65th Annual Meeting of The American Society of Human Genetics},
    October 9, 2015, Baltimore, MD.
  \item
    Gossmann A., Cao S., Wang Y.-P., Identification of Significant Genetic Variants via SLOPE, and its Extension to Group SLOPE;
    Contributed talk presented at the \textit{6th ACM Conference on Bioinformatics, Computational Biology, and Health Informatics},
    September 2015, Atlanta, GA.
\end{itemize}

%---

\section{SOFTWARE}

\begin{itemize}
  \item \verb!ThresholdoutAUC! -- R and Python code associated with the publication \cite{Gossmann2021-bp}. Project repository: \url{https://github.com/DIDSR/ThresholdoutAUC}.
  \item \verb!grpSLOPE! -- Group SLOPE (Group Sorted L1 Penalized Estimation) is a penalized linear regression method that is used for adaptive selection of groups of significant predictors in a high-dimensional linear model. This R package has been used to perform simulations and/or analyses of real genomic data as reported in \cite{Gossmann2017-yu, brzyski2016}. Project repository: \url{https://github.com/agisga/grpSLOPE}.\\
    CRAN: \url{https://cran.r-project.org/web/packages/grpSLOPE/index.html}.
  \item \verb!grpSLOPEMC! -- An extension package to the R package grpSLOPE, which contains additional Monte Carlo based methods implemented in R and C++ (interfaced to R via Rcpp). This R package has been used to perform simulation studies and analyses of real genomic data presented in \cite{Gossmann2017-yu}. Project repository: \url{https://github.com/agisga/grpSLOPEMC}.
  \item \verb!FDRcorrectedSCCA! -- Codes associated with the publication \cite{Gossmann2017-ln} with all methods and algorithms conveniently organized as functions in an R package. Project repository: \url{https://github.com/agisga/FDRcorrectedSCCA}.
  \item \verb!mixed_models! -- Fit statistical linear models with fixed and random effects in Ruby. It was created during Google Summer of Code 2015, and has been considered for deployment at \url{http://www.genenetwork.org}. Project repository: \url{https://github.com/agisga/mixed_models}. RubyGems: \url{https://rubygems.org/gems/mixed_models}.
  \item For my contributions to several other open source software projects see \url{https://github.com/agisga}.
\end{itemize}

%---

\section{EXTRA-CURRICULAR ACTIVITIES}

\begin{itemize}
  \item Elected {\it President} of the SIAM student chapter at Tulane University (Society for Industrial and Applied Mathematics), September 2014 -- September 2016.
  \item Organized the Graduate Student Colloquium at the Department of Mathematics, Tulane University, September 2014 -- September 2016.
  \item Participation in \emph{Google Summer of Code} as a student developer in 2015, and as a mentor in 2016 and 2017, working on scientific open source software.
  \item Participation in the middle school outreach program \emph{Girls in STEM at Tulane (GIST)}, November 2017.
\end{itemize}

%---

\section{HONORS \& AWARDS}

\begin{itemize}
  \item FDA/CDRH Excellence in Scientific Research Award for developing innovative approaches for reducing bias in artificial intelligence and machine learning testing protocols. June 2021.
  \item Finalist poster at the Tulane School of Science and Engineering Poster Days 2018.
  \item Reviewers’ Choice Abstract, 66th Annual Meeting of The American Society of Human Genetics, October 19, 2016, Vancouver, Canada.
  \item SIAM Student Chapter Certificate of Recognition, SIAM (Society for Industrial and Applied Mathematics), April 2015.
\end{itemize}

\vfill
\centerline{Updated: \today}

%----------------------------------------------------------------------------------------

\end{resume}

\end{document}
